tests/.eslintrc

```diff
@@ -0,0 +1,2 @@
+{ "rules": { "no-undef": "off" } }
```

Comment:
`jest` injects a bunch of globals, which I don't love. Is there an `eslintrc` rule directive to allow just a certain set of globals?

Reply:
Ooo, I found `eslint` has a `jest` environment already. Updated.